app/controllers/bulk_update/bulk_updates_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  class BulkUpdatesController < ApplicationController
+    before_action { check_for_provider }
```

Comment:
Not sure if this is correct. Only provider users can view the bulk update page? I know they're the only ones who can bulk recommend but for lead school users, should they see the bulk updates link at all? @andrewscrivener